tests/test_base.py

```diff
@@ -691,40 +691,39 @@ def test_base_index_access():
         h[0:10:20j, 0:5:10j, "hello", False, 5]
 
 
-class test_base_storage_proxy:
+class Test_base_storage_proxy:
```

Comment:
```suggestion
class TestBaseStorageProxy:
```
Classes should always be CamelCase. :)